drivebrain_core_impl/drivebrain_comms/src/CANComms.cpp

```diff
@@ -147,9 +147,9 @@ void comms::CANDriver::_handle_recv_CAN_frame(const struct can_frame &frame)
 std::shared_ptr<google::protobuf::Message> comms::CANDriver::_get_pb_msg_by_name(const std::string &name)
 {
     // Create a dynamic message factory
-
     std::shared_ptr<google::protobuf::Message> prototype_message;
-    const google::protobuf::Descriptor *desc = google::protobuf::DescriptorPool::generated_pool()->FindMessageTypeByName(name);
+
+    const google::protobuf::Descriptor *desc = google::protobuf::DescriptorPool::generated_pool()->FindMessageTypeByName("hytech." + name);
```

Comment:
should add a nullptr check here as we have already seen that this can cause a seg fault